### PR TITLE
Matter: restructure section on adding device

### PR DESCRIPTION
**Pull request by c0ffeeca7**
do not merge unless approved by marcelveldt.

## Proposed change

Matter: restructure section on adding device
- add subtitles to prereqs
- delete generic config flow. Adding the integration is part of the procedure
- move intro section about integration to the top of the page
- Fix Nest Hub product name

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/matter.markdown [hunk 1]
```diff
@@ -107,18 +104,29 @@ Note: The section below mentions third-party Thread border routers such as the N
...
+
 - Have either an Android or iPhone ready and Bluetooth enabled. For information why Bluetooth is required, refer to the section on [Bluetooth used during commissioning](#bluetooth-used-during-commissioning):
   - Android:
     - Have an Android phone (a full Android, not F-Droid).
     - Have the latest version of the Home Assistant Companion app installed.
-    - Have Google Home app installed on the Android.
+    - Have the Google Home app installed on the Android.
     - We are not going to add the new device to Google Home. The app is needed because Google included the Matter SDK there.
-    - If you are using {% term Thread %}: Make sure there is a Thread border router device (Nest Hub v2 or Nest Wi-Fi Pro) present in your home network.
+    - If you are using {% term Thread %}: Make sure there is a Thread border router device (Nest Hub (2nd Generation) or Nest Wi-Fi Pro) present in your home network.
```
Comment (agners) [inline source/_integrations/matter.markdown]: Nit: At least the official U.S. store and support resources use "Nest Hub (2nd gen)" (some with a capital G some not :sweat_smile: )


```suggestion
    - If you are using {% term Thread %}: Make sure there is a Thread border router device (Nest Hub (2nd Gen) or Nest Wi-Fi Pro) present in your home network.
```

File: source/_integrations/matter.markdown [hunk 2]
```diff
@@ -88,16 +86,15 @@ Running Matter on a Home Assistant Core installation is not supported.
...
 
-Note: The section below mentions third-party Thread border routers such as the Nest Hub v2 or the HomePod Mini. This doesn’t mean you have to add your devices to these ecosystems. Home Assistant only uses them to access the Thread radio network. The communication between the Home Assistant Matter controller and your Matter devices is encrypted. The Thread border router passes the data along. It cannot read its content.
+Note: The section below mentions third-party Thread border routers such as the Nest Hub (2nd Generation) or the HomePod Mini. This doesn’t mean you have to add your devices to these ecosystems. Home Assistant only uses them to access the Thread radio network. The communication between the Home Assistant Matter controller and your Matter devices is encrypted. The Thread border router passes the data along. It cannot read its content.
```
Comment (agners) [inline source/_integrations/matter.markdown]: Same as above.

```suggestion
Note: The section below mentions third-party Thread border routers such as the Nest Hub (2nd Gen) or the HomePod Mini. This doesn’t mean you have to add your devices to these ecosystems. Home Assistant only uses them to access the Thread radio network. The communication between the Home Assistant Matter controller and your Matter devices is encrypted. The Thread border router passes the data along. It cannot read its content.
```

File: source/_integrations/matter.markdown [hunk 3]
```diff
@@ -30,6 +30,8 @@ ha_integration_type: integration
 
 The Matter integration allows you to control Matter devices on your local Wi-Fi or {% term Thread %} network.
 
+For communicating with Matter devices, the Home Assistant integration runs its own "Matter controller" in a separate process which will be launched as an add-on. This add-on runs the controller software and connects your Matter network (called Fabric in technical terms) and Home Assistant. The Home Assistant Matter integration connects to this server via a WebSocket connection.
```
Comment (agners) [inline source/_integrations/matter.markdown]:
```suggestion
For communicating with Matter devices, the Home Assistant integration runs its own "Matter controller" in a separate Matter Server add-on. This add-on runs the controller software as a separate process and connects your Matter network (called Fabric in technical terms) and Home Assistant. The Home Assistant Matter integration connects to this server via a WebSocket connection.
```

File: source/_integrations/matter.markdown [hunk 4]
```diff
@@ -107,18 +104,29 @@ Note: The section below mentions third-party Thread border routers such as the N
     - If you are already running the Matter server in another add-on, in or a custom container:
       - Deselect the checkbox, then select **Submit**.
       - In the next step, provide the URL to your Matter server.
+
+#### Check logos on the device
+
+- On the device packaging, check for both the Matter logo and for either the Wi-Fi or the {% term Thread %} logo.
+- Check if the QR code is only on the packaging or if it is also on the device.
+  - If it is only on the packaging, snap a picture of the QR code and the device and store the image and the numerical code in a save place.
+  - If you lose the QR code and disconnect the device at some point, you won't be able to connect to that device again without the QR code.
```
Comment (agners) [inline source/_integrations/matter.markdown]: From the devices I've seen, most of them are on the device (at least Eve, Onvis plug, Nanoleaf). But some have in their documentation (Aqara  Door and Window Sensor P2 is an example).

I'd reword this a bit to:

```suggestion
- Check if the QR code or the numeric setup code is on the device.
  - If you reset your device you'll need the QR code *or* numeric setup code to {% term commission %} that device again! Without this information, commissioning won't be possible.
  - If the QR code or the numeric setup code is only in accompanied documentation, it is good practice to snap a picture of the QR code and/or numeric setup code as a backup, ideally along with the device for reference, and store the code in a safe place.
```

File: source/_integrations/matter.markdown [hunk 5]
```diff
@@ -107,18 +104,29 @@ Note: The section below mentions third-party Thread border routers such as the N
...
+
+#### Prepare Android or iPhone
+
 - Have either an Android or iPhone ready and Bluetooth enabled. For information why Bluetooth is required, refer to the section on [Bluetooth used during commissioning](#bluetooth-used-during-commissioning):
   - Android:
     - Have an Android phone (a full Android, not F-Droid).
     - Have the latest version of the Home Assistant Companion app installed.
-    - Have Google Home app installed on the Android.
+    - Have the Google Home app installed on the Android.
     - We are not going to add the new device to Google Home. The app is needed because Google included the Matter SDK there.
```
Comment (agners) [inline source/_integrations/matter.markdown]: I've double checked wiith @jpelgrom, this really should not be necessary.

There were reports from users which had problems in the past, but reinstalling the HA App should help too. Also that was early days, I doubt a bit that this is still a problem.

Suggestion: Let's see if this problem appears again now that Matter support on Android matured. If so, we can add a hint to Troubleshooting.

But I'd suggest to drop this prerequisites here.

```suggestion
```

/cc @marcelveldt

File: source/_integrations/matter.markdown [hunk 6]
```diff
@@ -257,7 +265,7 @@ NOTE for Android users: You need to follow the instructions at the bottom of the
 
 - Using Thread-based Matter devices in Home Assistant requires Home Assistant OS version 10 and above. Not using Home Assistant OS is at your own risk. We do provide some [documentation](https://github.com/home-assistant-libs/python-matter-server/blob/main/README.md) on how to run the Matter Server as a Docker container. The documentation includes a description of the host and networking requirements.
 
-- To use {% term Thread %} devices you will need a {% term Thread %} network with at least one Thread border router in your network nearby the {% term Thread %} device(s). Apple users, for example, need the Apple TV 4K or the HomePod Mini, while Google users need a Nest Hub V2. Use the Thread integration in Home Assistant to diagnose your {% term Thread %} network(s).
+- To use {% term Thread %} devices you will need a {% term Thread %} network with at least one Thread border router in your network nearby the {% term Thread %} device(s). Apple users, for example, need the Apple TV 4K or the HomePod Mini, while Google users need a Nest Hub (2nd Generation). Use the Thread integration in Home Assistant to diagnose your {% term Thread %} network(s).
```
Comment (agners) [inline source/_integrations/matter.markdown]: Same as above.

```suggestion
- To use {% term Thread %} devices you will need a {% term Thread %} network with at least one Thread border router in your network nearby the {% term Thread %} device(s). Apple users, for example, need the Apple TV 4K or the HomePod Mini, while Google users need a Nest Hub (2nd Gen). Use the Thread integration in Home Assistant to diagnose your {% term Thread %} network(s).
```

File: source/_integrations/matter.markdown [hunk 7]
```diff
@@ -30,6 +30,8 @@ ha_integration_type: integration
 
 The Matter integration allows you to control Matter devices on your local Wi-Fi or {% term Thread %} network.
 
+For communicating with Matter devices, the Home Assistant integration runs its own "Matter controller" in a separate Matter Server add-on. This add-on runs the controller software as a separate process and connects your Matter network (called Fabric in technical terms) and Home Assistant. The Home Assistant Matter integration connects to this server via a WebSocket connection.
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]:
```suggestion
For communicating with Matter devices, the Home Assistant integration runs its own "Matter controller" as add-on. This Matter Server add-on runs the controller software as a separate process and connects your Matter network (called Fabric in technical terms) and Home Assistant. The Home Assistant Matter integration connects to this server via a WebSocket connection.
```

File: source/_integrations/matter.markdown [hunk 8]
```diff
@@ -72,11 +74,7 @@ For devices where Home Assistant provides a native integration (with local API),
...
 Image taken from [this excellent article by The Verge](https://www.theverge.com/23165855/thread-smart-home-protocol-matter-apple-google-interview) about Matter that shows the landscape of Matter, {% term Thread %}, Border routers and bridges in a nice visualized way.
 
-{% include integrations/config_flow.md %}
-
-For communicating with Matter devices, the Home Assistant integration runs its own "Matter controller" in a separate process which will be launched as an add-on. This add-on runs the controller software and connects your Matter network (called Fabric in technical terms) and Home Assistant. The Home Assistant Matter integration connects to this server via a WebSocket connection.
-
-### Supported installation types
+## Supported installation types
 
 It is recommended to run the Matter add-on on Home Assistant OS. This is currently the best-supported option.
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]:
```suggestion
It is recommended to run the Matter add-on on Home Assistant OS. This is currently the only supported option.
```

Comment (marcelveldt) [reply source/_integrations/matter.markdown]: other installation types are without support and completely on your own risk

**Review by marcelveldt — APPROVED** [review]
nice!
